code/modules/cm_tech/techs/marine/tier2/orbital_ammo.dm

```diff
@@ -0,0 +1,58 @@
+/datum/tech/repeatable/ob
+	name = "OB Ammo"
+	desc = "Purchase orbital bombardment ammo."
+
+	required_points = 10
+	increase_per_purchase = 2
+
+	tier = /datum/tier/two
+
+	announce_name = "ALMAYER SPECIAL ASSETS AUTHORIZED"
+
+	var/type_to_give
+
+/datum/tech/repeatable/ob/on_unlock()
+	. = ..()
+	if(!type_to_give)
+		return
+
+	var/datum/supply_order/O = new /datum/supply_order()
+	O.ordernum = supply_controller.ordernum
+	supply_controller.ordernum++
+	O.object = supply_controller.supply_packs[type_to_give]
+	O.orderedby = MAIN_AI_SYSTEM
+
+	supply_controller.shoppinglist += O
+
+/datum/tech/repeatable/ob/he
+	name = "Additional OB projectiles - HE"
+	desc = "Highly explosive bombardment ammo, to be loaded into the orbital cannon."
+	icon_state = "ob_he"
+
+	announce_message = "Additional Orbital Bombardment ordnance (HE) have been delivered to Requisitions' ASRS."
```

Comment:
"ordnance" is singular so it should be "ordnance has been delivered" or "warheads have been delivered". Not what it currently is.